Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -441,6 +447,13 @@ private static void Clean(NugetPackageIdentifier package)
                         selectedDirectories.Add(directory.FullName);
                         break;
                     }
+                    else
+                    {
+                        string versionName = GetNuGetVersionName(directoryName);
+                        // Current API version is NET_Standard_2_0; NET_Standard_4_5 is required to support this NuGet package.
+                        Debug.LogWarning(string.Format("Current API version is {0}; {1} is required to support this NuGet package.", DotNetVersion, versionName));
```

Comment:
I think a warning is too strong here as it's expected that there would be other target frameworks that don't work with Unity and this code is determining which of all the options best fits the current Unity settings. NuGetForUnity has a verbose logging concept that I think is a better fit for this point in the code. [Here's where](https://github.com/GlitchEnzo/NuGetForUnity/blob/911f02a5dc4d730139478f1805dc6d2990c5e3af/Assets/NuGet/Editor/NugetHelper.cs#L449) we log what folder we're selecting.

Maybe what you're looking for is a warning (or maybe even an error) if there are available target frameworks, but none of them can work with Unity's current settings? Testing `libDirectories.Count > 0 && selectedDIrectories.Count == 0` [here](https://github.com/GlitchEnzo/NuGetForUnity/blob/911f02a5dc4d730139478f1805dc6d2990c5e3af/Assets/NuGet/Editor/NugetHelper.cs#L445) would give you that.

Reply:
Sure, this recommended is good. I fixed it already. ;D